blog/2024-02-08-year-of-agama.mdx

```diff
@@ -0,0 +1,106 @@
+
+Although we will keep delivering new versions of Agama at a relatively constant pace, we established
+two milestones as check points. The first milestone is expected around mid April, the second one
+is scheduled to mid July. Then we used those two milestones to group the next tasks we want to
+tackle.
+
+April's milestone should result in a revamped architecture for Agama which should not longer depend
+on Cockpit and in a more comprehensive user interface for configuring the storage setup. Both
+aspects are explained in depth in this blog post.
+
+July's milestone should bring mechanisms to make Agama more adaptable and many improvements to
+unattended installation, turning Agama into a worthy contender to AutoYaST.
+
+Let's dive into the improvements expected for the first milestone.
+
+## Architectural Changes {#architecture}
+
+So far, we have built Agama on top of the infrastructure provided by the
+[Cockpit project](https://cockpit-project.org/). That allowed us to bootstrap the project quickly
+without having to invest too much on aspects like authentication or serving files to the web
+interface. But after more than a year of Agama development we now have a clear view on how we want
+to do certain things, and Cockpit is starting to feel like a limiting factor.
+
+See more details at [this Github discussion](https://github.com/agama-project/agama/discussions/1000),
+but in short we concluded the small of amount of functionality we are getting out of Cockpit does
+not justify the strong dependency, especially now that Cockpit is adopting Python as an integral
+part of its runtime.
+
+So we will invest the following months in changing a bit the approach as described in the mentioned
+discussion at Github. That should unblock many paths to improve Agama in the near future.
+
+## A More Powerful User Interface for the Storage Proposal {#storage}
+
+The mentioned architectural changes are important for remote or unattended installation and also
+regarding integration of Agama into bigger solutions, but they may not be that visible for the
+casual user. That doesn't imply next months will be boring in the area of interactive installation.
+Quite the opposite, we plan many improvements in the Agama's proposal page that allows to tweak the
+storage configuration.
+
+The new interface aims to be easy enough for newcomers, as you can see in the following mock-up.
+But we know (open)SUSE users have big expectations in terms of customizing their setups. Thus,
+we updated the [document](https://github.com/agama-project/agama/blob/master/doc/storage_ui.md) that
+describes how the new interface will work and all the possibilities it will offer for those who
+decide to go beyond the initial proposal. If you want to check whether your basic needs would be
+covered, don't hesitate to take a look to the document and the extended mock-ups included on it.
+
+![Storage proposal](../static/img/blog/2024-02-08/storage.png)
```

Comment:
Just a though: I would like to take this migration as a turning point regard the _alt_ text we're adding to images. I mean, keep this as it is if it comes from a migration but would be nice to start using better text in future blog entries.